src/components/adaptive-cards/AdaptiveCard/AdaptiveCard.jsx

```diff
@@ -155,8 +161,15 @@ export default function AdaptiveCard({
     });
 
     setInputs(newInputs);
+    const result = !Object.values(newInputs).some((input) => input.error);
```

Comment:
this reads: if not (some inputs have error)
i would prefer: if all inputs don't have errors

hint: use array.every instead of array.some